app/src/main/res/values-nb-rNO/strings.xml

```diff
@@ -0,0 +1,126 @@
+    <string name="viewCardTitle">Vis kundekort</string>
+    <string name="scanCardBarcode">Skann kortets strekkode</string>
+    <string name="cardShortcut">Kort-snarvei</string>
+    <string name="noCardsMessage">Legg til et kort først</string>
+
+    <string name="barcodeImageDescription">Bilde av kortets strekkode</string>
+
+    <string name="noStoreError">Ingen butikk angitt</string>
+    <string name="noCardIdError">Ingen kort-ID innskrevet</string>
+    <string name="noCardExistsError">Kunne ikke finne kundekort</string>
+
+    <string name="cardIdFormat">%1$s: %2$s</string>
+    <string name="storeNameAndNoteFormat" translatable="false">%1$s - %2$s</string>
+
+    <string name="importExport">Import/eksport</string>
+    <string name="importName">Importer</string>
+    <string name="exportName">Eksporter</string>
+    <string name="importExportHelp">Sikkerhetskopiering av data lar deg flytte kortene til en annen enhet.</string>
+    <string name="importedFrom">Importert fra: %1$s</string>
+    <string name="exportedTo">Exportert til: %1$s</string>
+    <string name="fileMissing">Fil mangler: %1$s</string>
+    <string name="importSuccessfulTitle">Importert</string>
+    <string name="importFailedTitle">Kunne ikke importere</string>
+    <string name="importFailed">Klarte ikke å importere: %1$s</string>
+    <string name="exportSuccessfulTitle">Eksportert</string>
+    <string name="exportFailedTitle">Kunne ikke eksportere</string>
+    <string name="exportFailed">Klarte ikke å eksportere: %1$s</string>
+    <string name="importing">Importerer&#8230;</string>
+    <string name="exporting">Exporterer&#8230;</string>
+    <string name="noExternalStoragePermissionError">Kunne ikke importere eller eksportere kort uten tilgang til ekstern lagring.</string>
+    <string name="exportOptionExplanation">Data skrives til rotmappen i eksternt lagringsområde.</string>
+    <string name="importOptionFilesystemTitle">Importer fra filsystem</string>
+    <string name="importOptionFilesystemExplanation">Velg spesifikk fil fra filsystemet.</string>
+    <string name="importOptionFilesystemButton">Fra filsystem</string>
+    <string name="importOptionApplicationTitle">Brukt eksternt program</string>
+    <string name="importOptionApplicationExplanation">Bruk eksternt program som Nextcloud, eller din favorittfilbehandler til å åpne ei fil.</string>
+    <string name="importOptionApplicationButton">Bruk eksternt program</string>
+    <string name="importOptionFixedTitle">Importer fra eksporteringsområde</string>
+    <string name="importOptionFixedExplanation">Importer fra området i filsystemet eksporter skrives til.</string>
+    <string name="importOptionFixedButton">Bruk eksporteringsplassering</string>
+
+    <string name="about">About</string>
+    <string name="app_copyright_fmt">Copyright 2016-<xliff:g>%d</xliff:g> Branden Archer</string>
+    <string name="app_license">Lisensiert GPLv3+.</string>
+    <string name="about_title_fmt">Om</string>
```

Comment:
The build is failing because it says this string is not in the correct format. This string is being passed on argument (the name of the application) and needs to have %s in it. The original is:

About <xliff:g id="app_name">%s</xliff:g>

Maybe the following will work?


```suggestion
    <string name="about_title_fmt">Om <xliff:g id="app_name">%s</xliff:g></string>
```